Documentation/network/concepts/masquerading.rst

```diff
@@ -65,7 +65,7 @@ to determine which devices the program is running on:
 From the output above, the program is running on the ``eth0`` and ``eth1`` devices.
 
 
-The eBPF-based masquerading can masquerade packets of the following IPv4 L4 protocols:
+The eBPF-based masquerading can masquerade packets of the following L4 protocols:
```

Comment:
Out of scope for this PR, but someday I'd love to see a clearer definition of masquerading in the page intro. We just sort of roll right into using it to mean a particular behavior without directly defining what it means when we use it. The current state isn't bad; I only think it could be better.